packages/app-bridge/src/react/useTemplateChooser.spec.tsx

```diff
@@ -9,6 +9,7 @@ import { TemplateLegacy } from '../types/TemplateLegacy';
 import { AppBridgeBlock } from '../AppBridgeBlock';
 import { useTemplateChooser } from './useTemplateChooser';
 import { withAppBridgeBlockStubs } from '../tests/withAppBridgeBlockStubs';
+import { TemplateLegacyDummy } from '..';
```

Comment:
Avoid import from `".."` or `"."` or `"../.."`, it can produce side-effect

Reply:
Adjusted it.